src/sentry/conf/server.py

```diff
@@ -2492,6 +2493,8 @@ def build_cdc_postgres_init_db_volume(settings):
     KAFKA_INGEST_REPLAYS_RECORDINGS: {"cluster": "default"},
     # Metrics Testing Topics
     KAFKA_SNUBA_GENERICS_METRICS_CS: {"cluster": "default"},
+    # User Ip Events -- bucketed by region name
+    KAFKA_USER_IP_EVENTS: {"cluster": {"_local": "default"}},
```

Comment:
Couldn't we configure each region with the relevant cluster, or always use the same cluster name and have the host/ip of that cluster name resolve differently to the regional brokers?

Reply:
I definitely like the idea of using DNS for these mappings, we just need to make sure XDS will work with this approach (but it sounded like it would.

Reply:
That's a plausible approach, yes.  It wasn't yet clear as I was writing this how much we could rely on XDS or our service discovery mechanisms, or that the DNS names were consistent enough to ensure a simple mapping between region name -> endpoint.

Furthermore, for me to bake this assumption in does require generally greater complexity in existing kafka consumer logic.  My strategy is to `defer` for now.  I expliitly plan to be looking into the ops side of this equation and figuring out our path forward / maintaining changes in the face of improvements, but I didn't want to overload assumptions here from the git go.

Given that right now we do not but one region + developlment, would it be acceptable to continue down this path of least resistance until I can ascertain the larger picture?

Reply:
Going forward with this static configuration for now, until the story for how and if we can leverage  XDS or similar to make this more autoconfigurable.